unified-codes/apps/web/src/reducers/ExplorerReducer.ts

```diff
@@ -18,6 +18,22 @@ const initialState = (): IExplorerState => {
       rowsPerPage: 25,
       type: 'medicinal_product',
     },
+    searchBar: {
+      input: '',
+      label: 'Search description',
+    },
+    table: {
+      count: 1,
+      entities: [{ code: 'A', description: 'Drug A', type: 'medicinal_product' }],
+      orderBy: EEntityField.DESCRIPTION,
+      orderDesc: false,
+      rowsPerPage: 25,
+      page: 1,
+    },
+    toggleBar: {
+      buttonTypes: [EEntityType.DRUG, EEntityType.UNIT_OF_USE, EEntityType.OTHER],
+      buttonStates: { [EEntityType.DRUG]: true, [EEntityType.UNIT_OF_USE]: false, [EEntityType.OTHER]: false }
+    }
```

Comment:
Just temporary stuff to get the app compiling and displaying stuff. Still needs styles and reducers for updating state... but it's compiling and rendering!